src/components/IdentityRadioList.vue

```diff
@@ -0,0 +1,58 @@
+<template>
```

Comment:
Actually, this component is badly named: we decided to use "account" instead of " identity" and this component allows to select an account, so "AccountSwitcher" looks better.